src/mgr/DaemonServer.cc

```diff
@@ -1348,8 +1352,28 @@ bool DaemonServer::_handle_command(
        }
     }
 
-    if (r && (prefix == "osd destroy" ||
-	      prefix == "osd purge")) {
+    if (prefix == "osd safe-to-destroy") {
+      if (!r) {
+        ss << "OSD(s) " << osds << " are safe to destroy without reducing data"
+           << " durability.";
+        safe_to_destroy.swap(osds);
+      }
+      if (f) {
+        f->open_object_section("osd_status");
+        f->dump_stream("safe_to_destroy") << safe_to_destroy;
```

Comment:
dump these values as arrays, not a string of comma-separated values

Reply:
> dump these values as arrays, not a string of comma-separated values

@liewegas Fixed now